store/storetest/scheme_store.go

```diff
@@ -392,9 +392,8 @@ func testSchemeStoreDelete(t *testing.T, ss store.Store) {
 	sres4 := <-ss.Scheme().Delete(d4.Id)
 	assert.Nil(t, sres4.Err)
 
-	tres5 := <-ss.Team().Get(t4.Id)
-	assert.Nil(t, tres5.Err)
-	t5 := tres5.Data.(*model.Team)
+	t5, err := ss.Team().Get(t4.Id)
+	assert.Nil(t, err)
```

Comment:
nit: we can replace it with a `require.Nil` if not we can end up dereferencing a nil pointer.

Reply:
this change still needed

Reply:
@jespino sorry, forgot to push. Now it should be fine.